LDAR_Sim/src/methods/crew.py

```diff
@@ -112,6 +121,13 @@ def visit_site(self, site):
         Look for emissions at the chosen site.
         """
         m_name = self.config['label']
+        
+        # update locations of crew to site 
+        s_lon = float(site['lon'])
```

Comment:
Same with above, we could update position in mobile schedule

Reply:
Same with above